modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/Inner/Run/Run.tsx

```diff
@@ -23,9 +23,17 @@ import {searchUtil} from '../../../../../../util/search';
 import RunFormModal from './RunFormModal';
 import useRunActions from './useRunActions';
 
+interface BuildOutlet {
+	runId: Number | undefined;
```

Comment:
Number is different from number, you are passing a primitive type here

```suggestion
	runId?: number;
```

Reply:
https://www.tektutorialshub.com/typescript/typescript-number/

Reply:
Thanks Keven i know but i put this wrong.